examples/https/README.md

```diff
@@ -6,6 +6,11 @@ node ../../bin/webpack-dev-server.js --open --https
 
 A fake certificate is used to enable https.
 
+You can provide the following SSL options to override the fake certificate:
```

Comment:
Please provide this advanced documentation in [the wiki instead](https://github.com/webpack/docs/wiki/webpack-dev-server/_edit). The README only covers the most simple use case.